### ISSUE TITLE
Normalization (BN) layers in the stem part is frozen, Why?

### ISSUE THREAD

**Issue by chenchuanch** [ShuangLI59/person_search#87]
It is stated in the article that the batch normalization (BN) layers in the stem part should be fixed  as constant affine transformations, while keep the other BN layers as normal in the identification part.  In the train prototxt file the BN layer is frozen in stem part, and other not. Why? Can someone tell me reasons?

**Comment by Cysu** [comment]
@chenchuanch For the stem part, the batch size is 1. BN is not useful.
